Assets/Scripts/Managers/PauseMenuManager.cs

```diff
@@ -15,7 +15,15 @@ void Start()
 
     public void Continue()
     {
-        PlanetManager.Instance.Play(); //Uses KeypressManager.Play() same as pressing esc
+        if(PlanetManager.Instance != null )
+        {
+             PlanetManager.Instance.Play(); //Uses KeypressManager.Play() same as pressing esc
+        }
+        else if (KeypressManager.Instance != null)
+        {
+            KeypressManager.Instance.Play(); //Uses KeyPressManager.Play() same as pressing
+        }
+       
```

Comment:
Issue was that the script tried to reference the planetmanager while in the main scene. Added a fix to check if it is not null before either using planet manager or key press manager